appinventor/misc/mavendl/mavendl.py

```diff
@@ -0,0 +1,128 @@
+# Create mavendl to download AAR libs with transitive deps 
+# This stores the files in a "libs" directory relative to this script
+# Usage: mavendl.py <group>:<artifact>:<version>
```

Comment:
Which versions of python is this intended to work with?

Reply:
Not sure, but I use 3.10

Reply:
Ideally, should work for 3.8+